packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/ClientProvider.cs

```diff
@@ -40,29 +46,33 @@ protected override PropertyProvider[] BuildProperties()
             return [PipelineProperty];
         }
 
+        protected override FieldProvider[] BuildFields()
+        {
+            return [EndpointField];
+        }
         protected override ConstructorProvider[] BuildConstructors()
         {
             return
             [
                 new ConstructorProvider(
                     new ConstructorSignature(Type, $"{_inputClient.Description}", MethodSignatureModifiers.Public, []),
-                    new MethodBodyStatement[] { PipelineProperty.Assign(ClientPipelineSnippets.Create()).Terminate() },
+                    new MethodBodyStatements(
+                    [
+                        PipelineProperty.Assign(ClientPipelineSnippets.Create()).Terminate(),
+                        EndpointField.Assign(New.Instance(typeof(Uri), Literal("http://foo.com"))).Terminate()
```

Comment:
I think this is only a placeholder?

Reply:
yes its already replaced by Jorge today https://github.com/microsoft/typespec/blob/main/packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/UnbrandedTypeSpecClient.cs#L46